System.IO.Compression.TestData/System.IO.Compression.TestData.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageVersion>1.0.7-prerelease</PackageVersion>
```

Comment:
Why some of them are prerelease?

Reply:
Don't ask me 😂 this whole repo was a mess. I will bump all the versions with this PR.

Reply:
Fixed.